questions.json

```diff
@@ -165,4 +165,85 @@
         "option4": "Hearing",
         "correct_answer": "option1"
     }]
+    // Contributed by @justobioma
```

Comment:
As far as I know JSON doesn't support comments, so I think this will cause an error in program.

Edit: I tried adding this comment to questions.json in my system

> json.decoder.JSONDecodeError

This will throw JSONDecodeError. Please Consider removing the comment.
Your contribution is highly appreciated. :-)

Reply:
My bad! Will remove it asap